src/util/storageOptions.tsx

```diff
@@ -20,12 +21,17 @@ export const storageDrivers = [
     label: "ZFS",
     value: zfsDriver,
   },
+  {
+    label: "Ceph",
+    value: cephDriver,
+  },
 ];
 
 const storageDriverToSourceHelp: Record<string, string> = {
   dir: "Optional, path to an existing directory",
   lvm: "Optional, path to an existing block device, loop file or LVM volume group",
   zfs: "Optional, path to an existing block device, loop file or ZFS dataset/pool",
+  ceph: "Optional, existing OSD storage pool to use",
```

Comment:
This is wired, because there is also the `OSD pool name` under config. Seems like both do the same thing.
Do we really need both, or can we hide the source input for the ceph driver?

Reply:
now that you mentioned, it is weird since like you said it seems like they are doing the same thing. I recall Thomas mentioned that `source` should definitely be configurable, so let's rather remove the `osd_pool_name` config.

Reply:
This help text says "existing OSD storage pool to use", but the osd_pool_name is a custom name that will create the store. Maybe that is the difference and the reason both might be needed? Did you test the behaviour of the source field? will it also create a new storage pool if there is no pool with that name yet?

Reply:
![Screenshot from 2024-01-10 18-43-10](https://github.com/canonical/lxd-ui/assets/151511568/a6191811-5cf4-4807-864d-933ed41cbe7f)
![Screenshot from 2024-01-10 18-44-13](https://github.com/canonical/lxd-ui/assets/151511568/c376f97d-e76c-4d55-a673-e26d016300dc)

Reply:
this is resolved, changed help text for `source` config and removed `ceph.osd.pool_name`